### DISCUSSION TITLE
Box shadow not working

### DISCUSSION THREAD

**Discussion by SpencerMiller23** [tailwindlabs/tailwindcss#6827] [Help]
**What version of Tailwind CSS are you using?**

For example: v3.0.8

**What build tool (or framework if it abstracts the build tool) are you using?**

For example:  Next.js 12.0.7

**What version of Node.js are you using?**

For example: v16.13.0

**What browser are you using?**

For example: Chrome, Microsoft Edge

**What operating system are you using?**

For example: Windows

**Reproduction URL**

https://play.tailwindcss.com/gtdbhTPqNl

**Describe your issue**

I am trying to create a  mobile menu drawer toggle button similar to the one on the Star Wars website (https://www.starwars.com/). I am adding the box-shadow to some of the bars from the hamburger icon and they aren't appearing.

**Comment by adamwathan** [comment]
Hey! Those classes do work but the shadow you have chosen (shadow-xl) has a very large negative spread, and the browser won't render it on  elements below a certain size:

https://play.tailwindcss.com/cqWJQymI6t

You probably want a custom shadow for your specific situation.

**Reply by SpencerMiller23** [reply]
Hey thanks so much for the reply. By custom shadow, do you mean with regular CSS or is that something else with regards to Tailwind?

**Reply by adamwathan** [reply]
Hey! There are a few ways to do it, here's an example: https://tailwindcss.com/docs/box-shadow#using-custom-values
